Sources/Core/ObjectiveCDictionaryExtension.swift

```diff
@@ -12,8 +12,14 @@ extension ObjCModelRenderer {
     func renderGenerateDictionary() -> ObjCIR.Method {
         let dictionary = "dict"
 
-        let props = properties.map { (param, schemaObj) -> String in
-
+        let props = properties.filter { (_, schema) -> Bool in
+            switch schema.schema {
```

Comment:
Similar comment here about this filtering feeling a bit verbose.

Reply:
I'll have to try this later.